tests/_global.ts

```diff
@@ -4,11 +4,11 @@ import { IModule } from '../src';
 export function generateTestModule(): Partial<IModule> {
   const event = getTestEvent();
   return {
-    [`on_${event}`]: () => {},
+    [`on_${event}`]: () => void 0,
```

Comment:
should be undefined, right?
```suggestion
    [`on_${event}`]: () => undefined,
```